packages/functional-tests/pages/login.ts

```diff
@@ -355,6 +358,24 @@ export class LoginPage extends BaseLayout {
     return this.page.locator(selectors.SUBMIT).click();
   }
 
+  async waitForCWTSHeader() {
+    await this.page.waitForSelector(selectors.CWTS_HEADER, {
```

Comment:
I don't think our other tests with this...
```
    return this.page.isVisible(selectors.SYNC_CONNECTED_HEADER, {
      timeout: 100,
    });
```
... actually wait. It looks like `isVisible` executes immediately, even if I slapped a 5000 second timeout it wouldn't wait. Personally I think simply using `this.page.waitForSelector` in our functions and then a simple `await waitForCWTSHeader();` check is nice over checking for truthiness anyway; if it doesn't find the element the test fails.

Reply:
I'm onboard to make this change 👍🏽